frontend/src/Pages/StudentProfile.tsx

```diff
@@ -0,0 +1,287 @@
+                                                viewType="daily"
+                                            />
+                                        </ResponsiveContainer>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    {/* Cards */}
+                    <div className="w-[1/2] grid grid-cols-3 gap-4 mb-6 mt-6">
+                        <StatsCard
+                            title="Days Active"
+                            number={metrics.activity_engagement.total_active_days_monthly.toFixed(
+                                0
+                            )}
+                            label="Days Active This Month"
+                            tooltip="Total number of days resident has been active in UnlockedEd"
+                            useToLocaleString={false}
+                        />
+                        <StatsCard
+                            title="Average Activity Time"
+                            number={avgNumber}
+                            label={avgLabel}
+                            tooltip={avgToolTip}
+                            useToLocaleString={false}
+                        />
+                        <StatsCard
+                            title="Total Hours"
+                            number={weekNumber}
+                            label={weekLabel}
+                            tooltip={weekToolTip}
+                        />
+                    </div>
+                    {/* Tables */}
+                    <div className="grid grid-cols-2 gap-3 mb-6 mt-6">
+                        <div className="card pt-2 px-3">
+                            <div className="text-teal-4 text-center text-lg font-semibold">
+                                Top 5 Most Viewed Libraries
+                            </div>
+                            <table className="table-2 mb-4">
+                                <thead>
+                                    <tr className="grid-col-3">
+                                        <th className="justify-self-start">
+                                            Library Name
+                                        </th>
+                                        <th># Hours Watching</th>
+                                        <th className="justify-self-end">
+                                            Is Featured
+                                        </th>
+                                    </tr>
+                                </thead>
+                                <tbody className="flex flex-col gap-4 mt-4">
+                                    {metrics.top_libraries.map(
+                                        (items: OpenContentResponse) => {
+                                            return (
+                                                <tr key={items.content_id}>
```

Comment:
Can we make the rows clickable so that the user can navigate to the library if they click on them?